lib/decision_review/service.rb

```diff
@@ -29,24 +29,10 @@ class Service < Common::Client::Base
     # @return [DecisionReview::Responses::Response] Response object that includes the body,
     #                                               status, and schema validations.
     #
-    def post_higher_level_reviews(request_body)
+    def post_higher_level_reviews(body:, user:)
       with_monitoring_and_error_handling do
-        raw_response = perform(:post, 'higher_level_reviews', request_body)
-        DecisionReview::Responses::Response.new(raw_response.status, raw_response.body, 'intake_status')
-      end
-    end
-
-    ##
-    # Retrieve a Higher Level Review intake status.
-    #
-    # @param uuid [uuid] The intake uuid provided from the response of creating a new Higher Level Review
-    # @return [DecisionReview::Responses::Response] Response object that includes the body,
-    #                                               status, and schema avalidations.
-    #
-    def get_higher_level_reviews_intake_status(uuid)
```

Comment:
v0/intake_statuses_controller.rb still calls `get_higher_level_reviews_intake_status`

Reply:
good catch. i just removed the controller 👍 
(it's no longer an endpoint for Lighthouse)